server/flaskr/blueprints/organization.py

```
@@ -0,0 +1,92 @@
+        user = UserDataHandler.get_user_by_email(email)
+        if user is None:
+            OrganizationDataHandler.add_invite(orgId=orgId, email=email)
+
+            email_obj = EmailDrafter.create_organization_invitation_email(email, orgId, org_name, lang)
+            success &= EmailSender.send_email(email_obj)
+        else:
+            success &= UserDataHandler.add_user_to_org(user.id, orgId)
+    
+    if success:
+        return "", 200
+    else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)